tests/test_dakara_feeder.py

```diff
@@ -31,12 +95,15 @@ def test_feed(
         """
         # create the mocks
         mocked_dakara_server_class.return_value.get_songs.return_value = [
-            {"id": 0, "path": Path("directory_0/song_0.mp4")},
-            {"id": 1, "path": Path("directory_1/music_1.mp4")},
+            {"id": 0, "path": Path("directory_0/song_0.mp4").normpath()},
+            {"id": 1, "path": Path("directory_1/music_1.mp4").normpath()},
```

Comment:
I guess the normpath is for windows compatibility, tell me if I'm wrong.

Reply:
Yes. But there is a bit more elegant way: `Path("directory_0") / "song_0.mp4"`.